src/Apps/Consign/Routes/SubmissionFlow/Utils/validation.ts

```diff
@@ -7,20 +7,17 @@ export const artworkDetailsValidationSchema = yup.object().shape({
     .required(
       "Please select an artist from the list. Other artists cannot be submitted due to limited demand."
     ),
-  year: yup.string().required("Please enter the year").trim(),
+  year: yup.string().trim(),
   title: yup.string().required("Please enter the title").trim(),
-  materials: yup.string().required("Please enter materials").trim(),
-  rarity: yup
-    .string()
-    .required("Please select a rarity")
-    .test("isDefault", "", rarity => rarity !== "default"),
+  materials: yup.string().trim(),
+  rarity: yup.string().test("isDefault", "", rarity => rarity !== "default"),
   editionNumber: yup.string().when("rarity", {
     is: "limited edition",
-    then: yup.string().required("Please enter the edition number").trim(),
+    then: yup.string().trim(),
   }),
   editionSize: yup.string().when("rarity", {
     is: "limited edition",
-    then: yup.string().required("Please enter the edition size").trim(),
+    then: yup.string().trim(),
   }),
   height: yup.string().required("Please enter the height").trim(),
   width: yup.string().required("Please enter the width").trim(),
```

Comment:
@dblandin I've tried somehow to add feature flag toggling here but unable to do so, any suggestions? I've left it just as required for now.

Reply:
A bit verbose but maybe something like this?

```tsx
{
  height: (isSellFlowRequiredField ? yup.string() : yup.string().required("Please enter the height")).trim(),
  width: (isSellFlowRequiredField ? yup.string() : yup.string().required("Please enter the height")).trim()
}
```

Reply:
I tried but we're not able to add the feature flag in this file unless we import it or refactor it entirely. 
I guess we could export the feature flag from ArtworkDetailsForm, where it's also used...

Reply:
Ahh, I see. Yeah, in that case maybe we switch this helper to a generator function and update the few call sites that currently rely on `artworkDetailsValidationSchema`. Within the react component, we can check the feature flag and pass the boolean value in.

```ts

const baseArtworkDetailsValidationSchema = yup.object().shape({
  // snip...
  year: yup.string().trim(),
  // snip...
})

export const generateArtworkDetailsValidationSchema = (isSellFlowRequiredField: boolean = false) => {
  baseArtworkDetailsValidationSchema.concat(
    yup.object().shape({
      height: (isSellFlowRequiredField ? yup.string() : yup.string().required("Please enter the height")).trim(),
      width: (isSellFlowRequiredField ? yup.string() : yup.string().required("Please enter the height")).trim()
    })
  )
}
```

Reply:
Curious how come the feature flag isn't available? It should be available always within the codebase. There's a hook (which can't be used everywhere), but also the general `getENV`-backed fallback which should be accessible everywhere (client and server) - https://github.com/artsy/force/blob/6aefbee9677bdae053f6abd1828775584399079d/src/System/useFeatureFlag.tsx#L76

Reply:
Ah, that's a good reminder about `getFeatureFlag` function. Makes sense that the React hook variant doesn't work in this context but we should try this lower-level function 👍🏼